python/cookiecutter-django.md

```diff
@@ -124,8 +124,9 @@ In order to use pipenv and avoid confusions we are going to generate a Pipfile w
   - `requirements.txt` from the root folder of the project.
   - `requirements/base.txt`, `requirements/local.txt`, `requirements/production.txt` and the `requirements` folder.
 
-NOTE:
+NOTES:
 - Remember, when you install a new dependency, check if it will be needed at production or not. In the case that it won't be needed, add the flag `--dev` at the end of the installation command.
+- In the Pipfile, try to avoid the use of `"*"` for the library versions. For each library set the used version in the project. This is to avoid future compatibility errors.
```

Comment:
```suggestion
- In the Pipfile, try to avoid the use of `"*"` for the library versions. Set a specific version for each library in order to avoid future compatibility errors.
```